### PR TITLE
Sd 66 Roster page

### PR DESCRIPTION
**Pull request by kkatlopez**
please test on your machine!

### CODE REVIEW

**Comment by kkatlopez** [comment]
btw, images are still not in -- not sure how we wanna do that, but i'll create a Jira issue for it

File: client/src/components/RosterProfileLatest.js [hunk 1]
```diff
@@ -1,62 +1,96 @@
...
+          }
+        )
+    }
+
+    getLatestMeet(latestmeet, fullname) {
+      fetch("http://localhost:3001/meet_info")
+        .then(res => res.json())
+        .then(
+          (result) => {
+            var specific_result = result.find(x => (x.meetName === this.state.latestmeet[0] && x.meetStartDate === this.state.latestmeet[1]));
```
Comment (matthewyoungbar) [inline client/src/components/RosterProfileLatest.js:37]: If the find doesn't find anything, it returns `undefined` and crashes the app on line 40.

Comment (matthewyoungbar) [reply client/src/components/RosterProfileLatest.js:37]: The error happened a few times for me but not for every card...

**Review by matthewyoungbar — CHANGES_REQUESTED** [review]
Do you want to integrate pictures with this PR?

File: client/src/components/RosterProfileLatest.js (hunk 1 above)
Comment (kkatlopez) [reply client/src/components/RosterProfileLatest.js:37]: do you remember which cards?

**Comment by kkatlopez** [comment]
@matthewyoungbar how did you want to store images again? were we gonna scrape them or

**Comment by matthewyoungbar** [comment]
> @matthewyoungbar how did you want to store images again? were we gonna scrape them or

I thought were planning on putting them under credentials but we never fully decided

File: client/src/components/RosterProfileLatest.js (hunk 1 above)
Comment (kkatlopez) [reply client/src/components/RosterProfileLatest.js:41]: @matthewyoungbar fixed, please test

**Review by gwynethyuen — CHANGES_REQUESTED** [review]
- latest tab on specific swimmer page isn't scrollable
- is by event supposed to show stuff? nothing's working for me

**Review by matthewyoungbar — APPROVED** [review]

**Pull request merged**